plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPlugin.groovy

```diff
@@ -38,7 +40,17 @@ public class AndroidCommandPlugin implements Plugin<Project> {
         }
     }
 
-    static def defaultTask(Project project, String taskName, String description, Class<? extends AdbTask> taskType, Closure configuration) {
+    private configureInputScripts(AndroidCommandPluginExtension extension, Project project) {
+        extension.scripts.all { input ->
+            project.tasks.create(input.name, Input) {
+                group = 'adb script'
```

Comment:
I decided to have a brand new `group` called `adb script`. Suggestions are welcome.